mkdocs.yml

```diff
@@ -30,6 +30,15 @@ nav:
       - User data: developers/user-data.md
       - GitHub actions: developers/github-actions.md
       - MkDocs and markdown-link-validator: developers/mkdocs.md
+      - Code quality tools: 
+          - Overview: developers/code-quality-tools/code-quality-tools.md
+          - General formatting: developers/code-quality-tools/general-formatting.md
+          - Python formatting: developers/code-quality-tools/python-formatting.md
+          - Python docstring formatting: developers/code-quality-tools/python-docstring-formatting.md
+          - Python type checking: developers/code-quality-tools/python-type-checking.md
+          - Python linting: developers/code-quality-tools/python-linting.md
+          - Security checks: developers/code-quality-tools/security.md
+          - See also: developers/code-quality-tools/python-see-also.md
```

Comment:
see my comments above. It is not the purpose of our documentation to describe the python ecosystem. Our documentation should be about our code/product, and to some extent on how this code is written/checked/etc. But anything beyond that just convolutes the docs and makes them less accessible and relevant.